test/testinput/bufr_bufr_backend_satwnd_amv_goes_mpi4.yaml

```diff
@@ -0,0 +1,58 @@
+time window:
+  begin: "2018-04-14T21:00:00Z"
+  end: "2023-12-15T03:00:00Z"
+
```

Comment:
Additionally, I think you should add "  bound to include: begin"

Reply:
```
time window:
  begin:  # datetime in ISO format
  length: # duration in ISO format
  bound to include: end # which window bound is inclusive (options: begin, end (default)).
```
Since the end is inclusive already (default),  we should add `bound to include: begin`.
I will add it.  Thanks!